lib/src/config/config_file/client.example.toml

```diff
@@ -19,15 +19,10 @@
 # Multiple addresses can be specified.
 #
 # If `advertised_addresses` is not used, these addresses will also be advertised to peers.
-listen_addresses = [
-  "/ip4/0.0.0.0/tcp/8443/ws",
-  "/ip6/::/tcp/8443/ws",
-]
+listen_addresses = ["/ip4/0.0.0.0/tcp/8443/ws", "/ip6/::/tcp/8443/ws"]
```

Comment:
Please don't change this formatting.

Reply:
Can you share why?

Reply:
It's less readable. Is this auto-formatted or why does this show up in @ii-cruz's PRs always?

Reply:
Yes, this is the default formating.